addons/hr_holidays/static/src/scss/accrual_plan_level.scss

```diff
@@ -105,3 +105,15 @@ $o-hr-holidays-border-color: gray('300');
         }
     }
 }
+
+.modal-content {
+    .btn.o_btn_remove {
+        margin-left: 0px;
+        padding-left: 4px;
+        padding-right: 4px;
+    }
+    .btn.o_form_button_cancel {
+        padding-left: 4px;
+        padding-right: 4px;
+    }
+}
```

Comment:
This will be applied globally on all modals once hr_holidays is installed.

It should be encapsulated in a class that's specific for hr_holidays.

Reply:
Indeed, thanks @kba-odoo :-)
I have added a second commit to add a specific class to the buttons. I am not fan of the solution but extending FormViewDialog to create a custom one would require to override _OnOpenCreate since the dialog is created and opened inside the method